README.md

```diff
@@ -1,2 +1,44 @@
-# Building-with-Backgrounds-and-Gradients
-replicating an old apple page
+# Project Name
+
+> Building with Backgrounds and Gradients.
+
+![screenshot](./Demo.png)
+
+This Is Clone of Mac Products home page.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/kotoku-code/Building-with-Backgrounds-and-Gradients/tree/proto-type)
```

Comment:
This link is broken.
Don't forget when project does get approved to change the link to a working link. 😅